insonmnia/miner/overseer.go

```diff
@@ -173,6 +201,10 @@ func (o *overseer) collectStats() {
 			}
 			o.mu.Unlock()
 			for _, id := range ids {
+				if len(id) == 0 {
```

Comment:
Yeah, after the first collecting. But *while* the first collecting there are empty strings.